posthog/async_migrations/runner.py

```diff
@@ -191,10 +191,16 @@ def attempt_migration_rollback(migration_instance: AsyncMigration, force: bool =
         # while rolling back. under normal circumstances, the error is reserved to
         # things that happened during the migration itself
         if force:
+            last_error = f"Force rollback failed with error: {error}"
             update_async_migration(
-                migration_instance=migration_instance,
-                status=MigrationStatus.Errored,
-                last_error=f"Force rollback failed with error: {error}",
+                migration_instance=migration_instance, status=MigrationStatus.Errored, last_error=last_error,
+            )
+
+        if is_email_available():
```

Comment:
Lets add a way to opt out of the emails via an env, given that this will be slightly longer then maybe put it into a function so we can use it from both locations

Reply:
good suggestion, done

Reply:
should be `async_migrations_emails_enabled`